inbm/dispatcher-agent/fpm-template/etc/apparmor.d/usr.bin.inbm-dispatcher

```diff
@@ -232,4 +232,6 @@
   /usr/sbin/chroot rUx,
   /usr/** r,
   /usr/bin/fwupdate rUx,
+  /usr/bin/python3* rUx,
+  /home/root/SblFwUpdate.py rUx,
```

Comment:
firmware tools shouldn't be installed in /home/root -- should be in /usr/bin, /usr/sbin, /usr/local/bin, or /usr/local/sbin.

also, /root is usually the home directory for root, not /home/root